app/controllers/languages_controller.rb

```diff
@@ -6,6 +6,15 @@ def index
     @body_id = 'languages'
     @locales = Locale.all
 
+    @locales.each do |locale|
+      locale.articles_count = Article.live.published.where(locale: locale.abbreviation).count
```

Comment:
does this populate and persist the column? or do you need to do something like `locale.update!(articles_count: count)`

Reply:
It does not _populate and persist_. I was thinking we could do that in a separate PR as an `after_save` hook on `Article`, then remove this block altogether. 

We could also add a `.live` (or similar) scope on `Locale` to accomplish the same thing.